src/server/utils/issue-has-new-activity.ts

```diff
@@ -0,0 +1,27 @@
+import {userModel} from '../models/userModel';
+
+export async function issueHasNewActivity(
+    username: string, lastActivity: number, userLastViewed: number|null):
+    Promise<boolean> {
+  const userRecord = await userModel.getUserRecord(username);
```

Comment:
I'm not familiar with firestore and how its caching works. I assume this always does a read.

In that case, I would prefer `userRecord` to be passed in to avoid multiple reads of the same data. The logic to set up the feature can move into `userModel` or another module.